web/templates/settings/privacy_settings.hbs

```diff
@@ -0,0 +1,81 @@
+  {{/if}}>
+    {{#if for_realm_settings}}
+    <div class="subsection-header inline-block">
+        <h3 class="inline-block">{{t "Privacy settings"}}</h3>
+        {{> settings_save_discard_widget section_name="privacy-setting" show_only_indicator=false }}
+    </div>
+    {{else}}
+    <h3 class="inline-block">{{t "Privacy"}}</h3>
+    {{> settings_save_discard_widget section_name="privacy-setting" show_only_indicator=(not for_realm_settings) }}
+    {{/if}}
+
+    <div class="input-group">
+        {{> settings_checkbox
+          setting_name="send_private_typing_notifications"
+          is_checked=settings_object.send_private_typing_notifications
+          label=settings_label.send_private_typing_notifications
+          prefix=prefix
+          }}
+        {{> settings_checkbox
+          setting_name="send_stream_typing_notifications"
+          is_checked=settings_object.send_stream_typing_notifications
+          label=settings_label.send_stream_typing_notifications
+          prefix=prefix
+          }}
+        {{> settings_checkbox
+          setting_name="presence_enabled"
+          is_checked=settings_object.presence_enabled
+          label=settings_label.presence_enabled
+          label_parens_text=settings_label.presence_enabled_parens_text
+          help_link="/help/status-and-availability"
+          prefix=prefix
+          }}
+        {{#if for_realm_settings}}
+        {{> settings_checkbox
+          setting_name="send_read_receipts"
+          is_checked=settings_object.send_read_receipts
+          label=settings_label.send_read_receipts
+          prefix=prefix
+          help_link="/help/read-receipts"
+          }}
+        {{else}}
+        {{> settings_checkbox
+          setting_name="send_read_receipts"
+          is_checked=settings_object.send_read_receipts
+          label=settings_label.send_read_receipts
+          help_link="/help/read-receipts"
+          help_icon_tooltip_text=send_read_receipts_tooltip
+          hide_tooltip=realm.realm_enable_read_receipts
+          }}
+        {{> settings_checkbox
+          setting_name="allow_private_data_export"
+          is_checked=settings_object.allow_private_data_export
+          label=settings_label.allow_private_data_export
+          help_link="/help/export-your-organization#export-your-organization"
+          }}
+        {{/if}}
+    </div>
+
+    <div class="input-group">
+        <label {{#if for_realm_settings}}for="realm_email_address_visibility"
+          {{else}}for="user_email_address_visibility" {{/if}} class="settings-field-label">
+            {{#if for_realm_settings}}
+            {{t "Who can access user's email address"}}
+            {{else}}
+            {{t "Who can access your email address"}}
+            {{/if}}
+            {{> ../help_link_widget link="/help/configure-email-visibility" }}
+        </label>
+        <div {{#unless for_realm_settings}} id="user_email_address_dropdown_container"
```

Comment:
There is no need for this `unless` block. Only block for `disabled_setting_tooltip` class is enough.